docs/PythonInspections.md

```diff
@@ -2,21 +2,500 @@
 
 TODO: add descriptions and examples
 
-- PyUnusedLocalInspection
-- PyRedundantParenthesesInspection
-- PySimplifyBooleanCheckInspection
-- PyDefaultArgumentInspection
-- PyArgumentEqualDefaultInspection
-- PyAssignmentToLoopOrWithParameterInspection
-- PyAsyncCallInspection
-- PyAttributeOutsideInitInspection
-- PyAugmentAssignmentInspection
-- PyBroadExceptionInspection
-- PyByteLiteralInspection
-- PyCallingNonCallableInspection
-- PyComparisonWithNoneInspection
-- PyDataclassInspection
-- PyDictCreationInspection
+<details>
```

Comment:
I have not known about such a thing, but it is not obvious that I should click to see the example. Maybe it is better in future to have list of issues and hyperlink to place in README with related examples.